### PR TITLE
Test modernise forms

### PR DESCRIPTION
**Pull request by timja**
https://github.com/jenkinsci/jenkins/pull/5923

### CODE REVIEW

**Comment by timja** [comment]
Two tests failing that need checking cc @janfaracik 
```
pct-workflow-durable-task-step-weekly / reuseNodesWithSameLabelsInDifferentReorderedStages – org.jenkinsci.plugins.workflow.support.steps.ExecutorStepTest
pct-throttle-concurrents-weekly / clearConfiguredCategories – hudson.plugins.throttleconcurrents.ThrottleJobPropertyTest
```

**Comment by janfaracik** [comment]
> pct-workflow-durable-task-step-weekly

Awesome, thanks a lot :) I've just forked `throttle-concurrents-weekly` I'll take a look and see how it goes

**Comment by timja** [comment]
`mvn test -Dsome.package.Test#testname` Can be used on the command line when that sort of thing happens, you may just need a `mvn install -Pquick-build` First

**Comment by janfaracik** [comment]
I've had a look at the other failing test - no idea what's up with it sadly

**Comment by timja** [comment]
Didn't fail in the last run, must have flaked

**Comment by timja** [comment]
Filed https://issues.jenkins.io/browse/JENKINS-67466 for the hangs we keep getting on ci.jenkins.io

**Comment by timja** [comment]
@janfaracik for throttle concurrent builds did the test dependencies not resolve for you in intellij ? I had things like `assertThat` and `@Test` being red lined. Tests worked fine via maven but something was breaking the intellij config

**Comment by janfaracik** [comment]
> @janfaracik for throttle concurrent builds did the test dependencies not resolve for you in intellij ? I had things like `assertThat` and `@Test` being red lined. Tests worked fine via maven but something was breaking the intellij config

Yeah I had the same issue, did a File -> Invalidate Caches and that fixed it

**Comment by timja** [comment]
weird, I used maven on the command line which is my usual workaround, the tests in `jenkins-test` part of Jenkins core don't work for me at all.

CLI FTR:
`mvn test -Dmaven.surefire.debug -Djenkins.version=2.327-SNAPSHOT  -Dtest=hudson.plugins.throttleconcurrents.ThrottleJobPropertyTest#clearConfiguredCategories`

allows debugging and exploring test Jenkins etc all fine

**Comment by timja** [comment]
🎉

File: sample-plugin/pom.xml [hunk 1]
```diff
@@ -14,7 +14,7 @@
     <properties>
         <changelist>999999-SNAPSHOT</changelist>
         <bom>weekly</bom>
-        <jenkins.version>2.326</jenkins.version>
+        <jenkins.version>2.327-rc32062.cc50203b9c3e</jenkins.version>
```
Comment (timja) [inline sample-plugin/pom.xml:17]:
```suggestion
        <jenkins.version>2.328-rc32102.8b35b58eb85d</jenkins.version>
```

**Pull request closed without merging**